mapie/tests/test_multi_label_classification.py

```diff
@@ -0,0 +1,599 @@
+from typing import Any, Optional
+
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.compose import ColumnTransformer
+from sklearn.datasets import make_multilabel_classification
+from sklearn.impute import SimpleImputer
+from sklearn.linear_model import LogisticRegression
+from sklearn.multioutput import MultiOutputClassifier
+from sklearn.pipeline import Pipeline, make_pipeline
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.utils.validation import check_is_fitted
+from typing_extensions import TypedDict
+
+from mapie._typing import NDArray
+from mapie.multi_label_classification import MapieMultiLabelClassifier
+
+Params = TypedDict(
+    "Params",
+    {
+        "method": str,
+        "bound": Optional[str],
+        "random_state": Optional[int]
+    }
+)
+
+METHODS = ["crc", "rcps"]
+BOUNDS = ["wsr", "hoeffding", "bernstein"]
+
+WRONG_METHODS = ["rpcs", "rcr", "test", "", 1, 2.5, (1, 2)]
+WRONG_BOUNDS = ["wrs", "hoeff", "test", "", 1, 2.5, (1, 2)]
+
+
+STRATEGIES = {
+    "crc": (
+        Params(
+            method="crc",
+            bound=None,
+            random_state=None
```

Comment:
should you specify a random state to set the results?

Reply:
done